aspnetcore/migration/1x-to-2x/identity-2x.md

```diff
@@ -302,8 +302,8 @@ In 2.0 projects, import the `Microsoft.AspNetCore.Authentication` namespace, and
 
 There are two variations of Windows authentication:
 
-* The host only allows authenticated users. This variation isn't affected by the 2.0 changes.
-* The host allows both anonymous and authenticated users. This variation is affected by the 2.0 changes. For example, the app should allow anonymous users at the [IIS](xref:host-and-deploy/iis/index) or [HTTP.sys](xref:fundamentals/servers/httpsys) layer but authorize users at the controller level. In this scenario, set the default scheme in the `Startup.ConfigureServices` method.
+- The host only allows authenticated users. This variation isn't affected by the 2.0 changes.
```

Comment:
😕 ... We always use the `*` for lists. Was there an internal change on this? Like I mentioned before, sometimes things change internally, and I get updated on it later.

Reply:
Thanks for going through this. That whole article uses `-`, so I saw a warning for the use of `*` here. I just made it consistent, but I can go and fix the whole article to use `*`. I probably should've just done that anyway.

Reply:
Oh, ok. Nevermind then. It's perhaps not worth the trouble.

Reply:
> Oh, ok. Nevermind then. It's perhaps not worth the trouble.

I'll do it. I just bottled it when I realised I'd have to change the whole file, but I shouldn't have. Thanks for pointing it out.

Reply:
> fighting a bit of a losing battle 

It's really a GIANT doc set at this point. We figured that we'd update things like this as we go primarily on total overhauls. When this topic is finally reached by someone in the distant future, then I think it would be made consistent.

Reply:
There we go, sorted. It was easier than I expected. Once I changed a couple, markdownlint told me that `-` was a bad egg and allowed me to fix all violations in the file with one click.